fuzzers/binary_only/qemu_coverage/src/fuzzer.rs

```diff
@@ -175,7 +178,7 @@ pub fn fuzz() {
 
         let stack_ptr: GuestAddr = qemu.read_reg(Regs::Sp).unwrap();
 
-        let reset = |buf: &[u8], len: GuestReg| -> Result<(), QemuRWError> {
+        let reset = |qemu: Qemu, buf: &[u8], len: GuestReg| -> Result<(), QemuRWError> {
             unsafe {
                 let _ = qemu.write_mem(input_addr, buf);
```

Comment:
This line is a bit clunky (although I think it was actually me that wrote it)! Can we change to match the following lines:
```
qemu.write_mem(input_addr, buf)?;
```